mints/07.json

```diff
@@ -0,0 +1 @@
+{"name":"skill/valory/market_creation_manager_abci:0.1.0","description":"The scaffold skill is a scaffold for your own skill implementation.","code_uri":"ipfs://bafybeihhtzwffrfs2owanv627tx53gtaapeelhwfemyzdmkysl7hwskf5q","image":"ipfs://bafybeifdvzkhdqhtit57ala2xorlf6tkl3p7di7bsl64yvhwvr575mon3y","attributes":[{"trait_type":"version","value":"0.1.0"}]}
```

Comment:
"The scaffold skill is a scaffold for your own skill implementation." -> you need to update the metadata of this package in its yaml @jmoreira-valory